WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1477,6 +1479,13 @@ public void run() {
         }).start();
     }
 
+    private boolean isDiscardable() {
+        if (!PostUtils.isPublishable(mPost) && isNewPost()) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
The `isDiscardable` method can be simplified to this:
``` java
private boolean isDiscardable() {
    return !PostUtils.isPublishable(mPost) && isNewPost();
}
```

Reply:
nice! addressed in 3d06f66